src/css/base/typography.scss

```diff
@@ -3,9 +3,14 @@
 
 body {
   color: $color-base;
+  font-feature-settings: 'kern';
```

Comment:
1. I rely on autoprefixer for x.browser, but http://caniuse.com/#feat=font-feature
2. http://blog.typekit.com/2014/02/05/kerning-on-the-web/
3. It's progressive enhancement, so if it fails on old browsers, it's not critical
4. looks like it should actually be `font-feature-settings: 'kern' 1;` so I'll make that change.